packages/babel-plugin-proposal-class-properties/test/fixtures/private/static-self-field/input.js

```diff
@@ -0,0 +1,8 @@
+function wrapper(wc) {
+  return wc
+}
```

Comment:
Nit: we don't need to declare the `wrapper` function for `input`/`output` tests, since it's not executed anyway.